snapshots/chunk.go

```diff
@@ -161,3 +163,20 @@ func DrainChunks(chunks <-chan io.ReadCloser) {
 		_ = chunk.Close()
 	}
 }
+
+// ValidRestoreHeight will check height is valid for snapshot restore or not
+func ValidRestoreHeight(format uint32, height uint64) error {
+	if format != snapshottypes.CurrentFormat {
```

Comment:
I think we should increase the format number?

Reply:
Right, I forgot to mention this. I wonder if we should also change it for the snapshot extensions support?

Reply:
but `CurrentFormat` is static , may be we need to change the `CurrentFormat` is configurable

Reply:
snapshot extensions feature is backward compatible if the app doesn't actually register any extensions, so I think it's good there.

> but CurrentFormat is static , may be we need to change the CurrentFormat is configurable

Just bump the number when there's a breaking change in the format? it's just for the receiver to fail fast when receiving an unsupported format.

Reply:
> @gsk967 , could you go thorough the comments and resolve them?

done